components/datetime/src/provider/fields/symbols.rs

```diff
@@ -786,23 +784,23 @@ impl LengthType for TimeZone {
 #[allow(clippy::exhaustive_enums)] // used in data struct
 pub enum DecimalSecond {
```

Comment:
@sffc this enums is identical to `SubsecondDigits`?

Reply:
No. `DecimalSecond` refers to a field that is a second with both second and subsecond digits. If this is unclear, feel free to improve the docs. I can't think of a better name for it: a decimal is a number that supports both an integer and a fraction.